tests/Doxyfile

```diff
@@ -656,6 +656,7 @@ WARN_LOGFILE           =
 INPUT                  = ../Bootloader/firmware/src \
                          ../common \
                          ../firmware/src \
+                         ../firmware/src/system_config/ethernet_sk2/system_settings.h \
```

Comment:
Sounds good, and avoids duplication of Doxygen, I was going to suggest something in that direction. If we go to Autotools, can we then just include the relevant board config depending on what you pick during configure.